relay/polkadot/src/lib.rs

```diff
@@ -708,7 +708,12 @@ impl pallet_staking::Config for Runtime {
 	type EventListeners = (NominationPools, DelegatedStaking);
 	type DisablingStrategy = pallet_staking::UpToLimitDisablingStrategy;
 	type WeightInfo = weights::pallet_staking::WeightInfo<Runtime>;
-	type Filter = pallet_nomination_pools::AllPoolMembers<Runtime>;
+	// TODO: this will come back later (stable2412/stable25XY)
+	// type Filter = pallet_nomination_pools::AllPoolMembers<Runtime>;
+
+	fn filter(who: &AccountId) -> bool {
+		pallet_nomination_pools::AllPoolMembers::<Runtime>::contains(who)
+	}
```

Comment:
👍

Reply:
Would be nice to write a small test for this ;)

Reply:
> Would be nice to write a small test for this ;)

I am checking https://github.com/paritytech/polkadot-sdk/compare/stable2409...bkchr-fix-the-thing and there were several pallet unit tests added:
`existing_pool_member_cannot_stake`
`stakers_cannot_join_pool`
`restricted_accounts_cannot_join`
`restricted_accounts_can_only_withdraw`
`permissionless_withdraw_overstake`

What should this small test cover/do? Some pseudo-code is ok:
```
// prepare account
// create pool
// accounts join the pool 

// <do-something-else>

// check something1 
// check something2
```

@Ank4n Could you please take care of this? Or at least, write down what needs to be tested, and I can write a test at runtimes.

Reply:
https://github.com/bkontur/runtimes/pull/11

Reply:
@bkchr tests are added (https://github.com/polkadot-fellows/runtimes/pull/623/commits/de119a3d7b947dda6b3c289842fe2c2434437723) by @Ank4n